joblib/_parallel_backends.py

```diff
@@ -295,7 +295,7 @@ def __init__(self, batch):
         # arguments in memory
         self.results = batch()
 
-    def get(self):
+    def get(self, timeout=None):
```

Comment:
What's the point of adding the timeout parameter, since you are not using it in the method?